cmd/describeCluster.go

```diff
@@ -0,0 +1,112 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	api "github.com/libopenstorage/openstorage-sdk-clients/sdk/golang"
+	"github.com/portworx/px/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+var describeClusterCmd *cobra.Command
+
+var _ = RegisterCommandVar(func() {
+	// describeClusterCmd represents the describeCluster command
+	describeClusterCmd = &cobra.Command{
+		Use:     "cluster",
+		Short:   "Describe a Portworx cluster",
+		Long:    "Show detailed information of Portworx cluster",
+		Example: `$ px describe cluster node-id`,
```

Comment:
Are we taking node-id as an input from user?

Reply:
I don't think this command should take a node id, instead we should have a `describe node <id>` command separate from this one. This is just the "cluster" information.

Reply:
Agree, corrected.